web_app/db/crud.py

```diff
@@ -827,6 +827,15 @@ def allow_notification(self, telegram_id: int) -> bool:
             user.is_allowed_notification = True
             session.commit()
             return True
+        
+    def is_allowed_notification(self, wallet_id: str = None) -> bool:
+        """
+        Returns true or false if a telegram user allowed notification.
+
+        Args:
+            wallet_id: Wallet ID of the user.
+        """
+        return self.is_allowed_notification
```

Comment:
You need to fetch TelegramUser and after then just fetch is_allowed_notification